packages/core/src/utils.ts

```diff
@@ -120,6 +122,30 @@ export class Utils {
     const version = cid.version
     await ipfsApi.block.put(block, { format, mhtype, version, signal })
   }
+
+  /**
+   * Attempts to load CACAO capability from IPFS if present in protected header of commit
+   * @param commit the commit to load the capability for
+   * @param dispatcher instance of dispatcher to load from IPFS
+   * @returns a Cacao capability object if found, else null
+   */
+  static async extractCapability(commit: any, dispatcher: Dispatcher): Promise<Cacao | undefined> {
+    if (commit.signatures && commit.signatures.length > 0) {
```

Comment:
Can do an early return here using the inverse of this condition, and avoid an extra indentation layer.  Early returns can do a lot to improve readability and reduce cognitive overhead when reading long functions like this